osu.Game/Beatmaps/BeatmapSetOnlineInfo.cs

```diff
@@ -12,11 +15,19 @@ namespace osu.Game.Beatmaps
     /// </summary>
     public class BeatmapSetOnlineInfo
     {
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; set; }

```

Comment:
why is this added (and why is this class modified)? currently this stuff isn't stored in the database

Reply:
Yes, there is actually no need to modify this class. It seems that I overdid it.